05-Organizing-and-Executing-Simulations/02-Executing-Simulations.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>Since <code>compressed_step</code>  will always be greater than zero we can use <code>FlowProject.post.true("compressed_step")</code>  here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>